src/app/build/build.component.html

```diff
@@ -14,201 +20,120 @@ <h1>
-            'active': build.worker.status == 'Pending'}">
-            <div class="success" ng-show="build.worker.status == 'Succeeded'">
-              <i class="icon ion-md-checkmark-circle"></i> {{ build.worker.status }}</div>
-            <!-- Pending API hookup:
-            <div class="failed ng-hide" ng-show="build.worker.status == 'Failed'"><i class="icon ion-md-close-circle"></i> Failed</div>
-            <div class="active ng-hide" ng-show="build.worker.status == 'Pending'"><img src="assets/images/rolling.svg"> In Progress</div>
-            -->
+          <div class="build-status">
+            <div class="unknown" *ngIf="!build.worker?.status"><i class="icon ion-md-radio-button-off"></i>  Log Unavailable</div>
+
+            <div class="pending" *ngIf="build.worker?.status == 'Pending'"><div class="spinner"></div> Initializing...</div>
+            <div class="running" *ngIf="build.worker?.status == 'Running'"><div class="spinner"></div> Build started...</div>
+
+            <div class="success" *ngIf="build.worker?.status == 'Succeeded'"><i class="icon ion-md-checkmark-circle"></i> {{ build.worker.status }}</div>
+            <div class="failed" *ngIf="build.worker?.status == 'Failed'"><i class="icon ion-md-close-circle"></i> {{ build.worker.status }}</div>
           </div>
 
-          <div class="grid-block horizontal build-metadata">
-            <div class="grid-content small-6 text-left">Started at {{ worker.start_time }} </div>
-            <div class="grid-content small-6 text-right">Finished at {{ worker.end_time }}</div>
+          <div class="row build-metadata">
+            <div class="small-6 text-left columns">Started at {{ build.worker.start_time }} </div>
+            <div class="small-6 text-right columns">Finished at {{ build.worker.end_time }}</div>
           </div>
 
-          <div class="horizontal build-chart build-chart-1" gantt="" data="data" ng-init="showChart = true" ng-hide="!showChart" ng-controller="jobsController">
+          <div class="horizontal build-chart build-chart-1">
 
             <div class="task task-0">
               <a href="">
                 <span title="git">
-                  <i class="icon logo-github"></i>
+                  <i class="icon ion-logo-github"></i>
                 </span>
               </a>
             </div>
 
-            <!-- ngRepeat: job in jobs -->
-            <div class="task task-1" ng-repeat="job in jobs">
-              <a ng-click="gotoElement('dind-1514014311816-master')">
-                <span title="job.name" class="ng-binding">JOB NAME</span>
-              </a>
-            </div>
-            <!-- end ngRepeat: job in jobs -->
-
-            <em></em>
-            <em></em>
-            <em></em>
-            <em></em>
-            <em></em>
+            <span class="task task-1"><a href="">{{ build?.type }}</a></span>
+            <span *ngFor="let job of jobs" class="task task-2"><a href="">{{ job.name }}</a></span>
```

Comment:
Because of the `href` here, when clicked, it will redirect to the home page - maybe open the logs of the respective job?

<img width="725" alt="view4" src="https://user-images.githubusercontent.com/13103165/42513165-ec7df794-845e-11e8-9916-ed962d483cbe.PNG">